packages/legend-query/src/stores/__tests__/QueryBuilder_LambdaProcessingRoundtrip.test.ts

```diff
@@ -62,6 +63,11 @@ const m2mCtx = {
 
 const cases: RoundtripTestCase[] = [
   ['Simple all() function', relationalCtx, TEST_DATA__simpleAllFunc],
+  [
+    'Simple all() function with parameter',
+    relationalCtx,
```

Comment:
This example is not of a milestoning class. Could we do a roundtrip test of a milestoning class?